temporian/implementation/numpy/data/plotter.py

```diff
@@ -46,10 +45,10 @@
 # "indexes" argument in tp.plot.
 InputIndex = Optional[

```

Comment:
In typing, what about having  both "InputIndex" (possible under a different name) for a user input and a normalized IndexKey (which is a tuple of int and bytes)?

Reply:
love it! done